libs/ui-toolkit/src/components/splash/splash.tsx

```diff
@@ -7,7 +7,7 @@ export interface SplashProps {
 
 export const Splash = ({ children }: SplashProps) => {
   const splashClasses = classNames(
-    'w-full h-full',
+    'w-full h-full text-xs text-center text-gray-200',
```

Comment:
aligning style of ag-grid overlay and splash classes

Reply:
Have you checked both light and dark themes here?